bukkit/compat/paper/src/main/java/com/discordsrv/bukkit/ban/PaperBanModule.java

```diff
@@ -45,14 +46,19 @@
 import java.util.concurrent.CompletableFuture;
 
 @ApiStatus.AvailableSince("Paper 1.21.1")
-public class PaperBanModule extends AbstractModule<BukkitDiscordSRV> implements Listener, PunishmentModule.Bans {
+public class PaperBanModule extends AbstractBukkitListener<PlayerKickEvent> implements PunishmentModule.Bans {
 
     public PaperBanModule(BukkitDiscordSRV discordSRV) {
-        super(discordSRV);
+        super(discordSRV, new NamedLogger(discordSRV, "BAN_MODULE"));
     }
 
     @EventHandler(priority = EventPriority.MONITOR, ignoreCancelled = true)
     public void onPlayerKick(PlayerKickEvent event) {
+        handleEvent(event);
+    }
+
+    @Override
+    protected void handleEvent(PlayerKickEvent event, Void __) {
         switch (event.getCause()) {
```

Comment:
This switch was intended to return if it wasn't caused by a ban, right now getting kicked will trigger ban sync to ban the user in Discord

Reply:
Only if the ban actually exists though? Either way I will move that inside the switch statement. Must have forgotten to.